api/RTCIceCandidatePairStats.json

```diff
@@ -1295,10 +1295,10 @@
               "version_added": null
             },
             "safari": {
-              "version_added": null
+              "version_added": false
```

Comment:
This (`retransmissionsReceived`) was also in https://github.com/WebKit/WebKit/commit/ed457da32eb3b03501a8741588c14245e4974f6d but I can't see it in any WebKit IDL file, so I guess it isn't supported.